README.md

```diff
@@ -57,3 +59,14 @@ This *should* play nice with a coexisting full installation of VirtualBox, but t
 The software itself consists of an auto-start background service.
 
 The installer also adds a firewall rule to allow all local subnets to connect to the service; this firewall rule can be tweaked to fine tune access control.
+
+## USBIP on WSL 2
+
+Currently WSL 2 does not support USB devices by default. A workaround to this limitation is to use usbip. For instructions on how to setup a Linux usbip client can be found [here](WSL_USBIP.md).
```

Comment:
Maybe you want the WSL specific instructions in the Wiki, as I can imagine they might change more frequently... But I'm also good with a README in the repo.

Reply:
Sure thing. I'll add WSL specific instructions in the Wiki